root/client/src/components/table/table.js

```diff
@@ -117,12 +101,58 @@ export const Table = ({ pollAvailabilities, setPollAvailabilities, duration, sta
         6: new Date().setTime(nextDayOfWeek.getTime() + 1000 * 60 * 60 * 24 * 6),
       }
       setWeek(newStateWeek);
-      setStartDay(nextDayOfWeek[Symbol.toPrimitive]('number'));
-      setEndDay(new Date().setTime(nextDayOfWeek.getTime() + 1000 * 60 * 60 * 24 * 6));
     }
 
   }
 
+  const handlePrevDay = (e) => {
+    e.preventDefault();
+    const previousDay = week[0] - (24 * 60 * 60 * 1000);
+    const startDateNum = new Date(`${startDate}T12:00:00.000Z`)[Symbol.toPrimitive]('number');
+
+    if (previousDay < startDateNum) {
+      return;
+    } else {
+      const newWeek = {
+        0: previousDay
+      }
+
+      const endDateNum = new Date(`${endDate}T12:00:00.000Z`)[Symbol.toPrimitive]('number');
+      for (let i = 1; i < 7; i++) {
+        const nextDayWeek = newWeek[0] + (24 * 60 * 60 * 1000 * i);
+        if (nextDayWeek > endDateNum) {
+          break;
+        } else {
+          newWeek[i] = nextDayWeek;
+        }
+      }
+      setWeek(newWeek);
+    }
+  }
+
+  const handleNextDay = (e) => {
```

Comment:
i've seen this logic a few times now. might be worth just making it a util function and referencing it where necessary

Reply:
The handleNextDay should only exist in one file, the table.js file. Let me know if i'm incorrect.